sound/soc/intel/boards/bytcht_es8316.c

```diff
@@ -605,10 +606,15 @@ static int snd_byt_cht_es8316_mc_probe(struct platform_device *pdev)
 	if (ret) {
 		gpiod_put(priv->speaker_en_gpio);
 		dev_err(dev, "snd_soc_register_card failed: %d\n", ret);
-		return ret;
+		goto err;
 	}
 	platform_set_drvdata(pdev, &byt_cht_es8316_card);
 	return 0;
+
+err:
+	put_device(priv->codec_dev);
+
```

Comment:
Seems redundant blank line (looking around on the style).

Reply:
ack